packages/location/tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "extends": "../../tsconfig.json",
   "include": ["src", "types"],
   "compilerOptions": {
-    "module": "esnext",
```

Comment:
wondering what was the motivation behind removing this?

Reply:
Fair enough I only ask because in the packages it's esnext and at the top level it's es2020. Just wanted to make sure this is a conscious change

Reply:
The module config is actually `exnext` on the top level :) 
https://github.com/contentful/field-editors/blob/5bbca863106912d6db500943b9bf77983059f29d/tsconfig.json#L5